fractal.py

```diff
@@ -124,15 +135,41 @@ def generate_fractal(model, c=None, size=pair_reader(int)(DEFAULT_SIZE),
   side = max(width, height)
   deltax = (side - width) / 2 # Centralize
   deltay = (side - height) / 2
-  img = pylab.array([
-    [func((2 * (col + deltax) / (side - 1) - 1) / zoom + cx,
-          (2 * (height - row + deltay) / (side - 1) - 1) / zoom + cy)
-      for col in range(width)]
-    for row in range(height)
-  ])
-
+  
+  # Start a timer
+  start = time.time()
+
+  # Create a pool of workers
+  numProcs = multiprocessing.cpu_count()
+  pool = multiprocessing.Pool(numProcs)
```

Comment:
Should be `num_procs`, as the code never uses lowerCamelCase. Not needed as a variable unless you print such information afterwards avoiding the private `pool._processes`.